--- a/Makefile
+++ b/Makefile
@@ -68,15 +68,15 @@
 # Install/uninstall
 
 install:
-	install -d $(DESTDIR)$(BINDIR)
-	install -m 755 ascii $(DESTDIR)$(BINDIR)/ascii
-	install -d $(DESTDIR)$(MANDIR)
-	install -d $(DESTDIR)$(MANDIR)/man1
-	install ascii.1 $(DESTDIR)$(MANDIR)/man1/ascii.1
+	install -d $(DESTDIR)$(PREFIX)$(BINDIR)
+	install -m 755 ascii $(DESTDIR)$(PREFIX)$(BINDIR)/ascii
+	install -d $(DESTDIR)$(PREFIX)$(MANDIR)
+	install -d $(DESTDIR)$(PREFIX)$(MANDIR)/man1
+	install ascii.1 $(DESTDIR)$(PREFIX)$(MANDIR)/man1/ascii.1
 
 uninstall:
-	rm -f $(DESTDIR)$(BINDIR)/ascii
-	rm -f $(DESTDIR)$(MANDIR)/man1/ascii.1
+	rm -f $(DESTDIR)$(PREFIX)$(BINDIR)/ascii
+	rm -f $(DESTDIR)$(PREFIX)$(MANDIR)/man1/ascii.1
 
 # Export
